api/src/routes/files/validation.ts

```diff
@@ -0,0 +1,72 @@
+import { z } from 'zod';
+
+export const FileNameParamSchema = z.object({
+	filename: z.string('Invalid file name!')
+});
+
+export type FileNameParam = z.infer<typeof FileNameParamSchema>;
+
+export const AllowedFileMimeTypesOptions = ['image/png', 'image/jpeg', 'image/jpg'] as const;
```

Comment:
```suggestion
export const AllowedFileMimeTypesOptions = ['image/png', 'image/jpeg'] as const;
```

The mime type is `image/jpeg`, it applies to two extensions: `jpg` and `jpeg`

Reply:
Thank you for sharing, Good to know. I thought they were separate.